tuya.cpp

```diff
@@ -434,11 +542,29 @@ void DeRestPluginPrivate::handleTuyaClusterIndication(const deCONZ::ApsDataIndic
                     }
                 }
                 break;
+                case 0x0210: // Thermostat heatsetpoint for moe
+                {
+                    qint16 temp = ((qint16)(data & 0xFFFF)) * 100;
```

Comment:
Please always use C++ named casts instead of unsafe C style casts. These will allow us better error checking at compile time.

```
qint16 temp = (static_cast<qint16>(data & 0xFFFF)) * 100;
```

See: https://github.com/dresden-elektronik/deconz-rest-plugin/pull/3423/files#r523413812

Reply:
Done